### PR TITLE
[Backport Foxy] Update Quality Declaration to reflect QL 2 (#160).

### PR DESCRIPTION
**Pull request by ahcorde**
This PR update the QD in Foxy. It related with with other PR https://github.com/ros/class_loader/pull/168

### CODE REVIEW

File: QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -138,21 +140,21 @@ It also has several test dependencies, which do not affect the resulting quality
 
 The `console_bridge_vendor` package provides a wrapper around `console_bridge`, providing nothing but a dependency on `console_bridge` on some systems. On others, it provides an [ExternalProject](https://cmake.org/cmake/help/latest/module/ExternalProject.html) build of `console_bridge`.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/console_bridge_vendor/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/console_bridge_vendor/blob/master/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md]: These links should point to `foxy` branches

Comment (ahcorde) [reply QUALITY_DECLARATION.md]: Done @chapulina https://github.com/ros/class_loader/pull/174/commits/6c21f4f64b2987c7cb1611d9c536b9fa5f6a449b

File: QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -138,21 +140,21 @@ It also has several test dependencies, which do not affect the resulting quality
 
 The `console_bridge_vendor` package provides a wrapper around `console_bridge`, providing nothing but a dependency on `console_bridge` on some systems. On others, it provides an [ExternalProject](https://cmake.org/cmake/help/latest/module/ExternalProject.html) build of `console_bridge`.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/console_bridge_vendor/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/console_bridge_vendor/blob/foxy/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md]: This is level 1

File: QUALITY_DECLARATION.md [hunk 3]
```diff
@@ -138,21 +140,21 @@ It also has several test dependencies, which do not affect the resulting quality
...
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcpputils/blob/foxy/QUALITY_DECLARATION.md).
 
 ### Direct Runtime non-ROS Dependency [5.iii]
 
 #### `libconsole-bridge-dev`
 
 The [libconsole-bridge-dev](https://github.com/ros/console_bridge/) is a ROS-independent, pure CMake (i.e. non-catkin and non-rosbuild package) that provides logging calls that mirror those found in rosconsole, but for applications that are not necessarily using ROS.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros/console_bridge/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros/console_bridge/blob/foxy/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md]: This repo doesn't have a `foxy` branch, I think we need to use `master`.

Also, it's QL1 already.

**Review by chapulina — COMMENTED** [review]
Just a couple more details

File: QUALITY_DECLARATION.md (hunk 3 above)
Comment (ahcorde) [reply QUALITY_DECLARATION.md]: updated

File: QUALITY_DECLARATION.md [hunk 4]
```diff
@@ -138,21 +140,21 @@ It also has several test dependencies, which do not affect the resulting quality
...
+It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcpputils/blob/foxy/QUALITY_DECLARATION.md).
 
 ### Direct Runtime non-ROS Dependency [5.iii]
 
 #### `libconsole-bridge-dev`
 
 The [libconsole-bridge-dev](https://github.com/ros/console_bridge/) is a ROS-independent, pure CMake (i.e. non-catkin and non-rosbuild package) that provides logging calls that mirror those found in rosconsole, but for applications that are not necessarily using ROS.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros/console_bridge/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 1**, see its [Quality Declaration document](https://github.com/ros/console_bridge/blob/foxy/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md]: The QD link is still nonexistent
```suggestion
It is **Quality Level 1**, see its [Quality Declaration document](https://github.com/ros/console_bridge/blob/master/QUALITY_DECLARATION.md).
```

**Comment by ahcorde** [comment]
friendly ping @chapulina

**Review by chapulina — APPROVED** [review]

**Pull request merged**